frontend/locales/en.json

```diff
@@ -362,7 +362,7 @@
           "attestations_target": "Target",
           "efficiency": "Efficiency",
           "luck": "Luck",
-          "missed_rewards": "Missed rewards",
+          "missed_rewards": " ",
```

Comment:
you are not removing `missed rewards` from the `ui` you are only `removing the language key`
<img width="977" alt="Bildschirmfoto 2024-11-04 um 08 39 29" src="https://github.com/user-attachments/assets/2f422dad-00b4-40ea-95c0-4814deb865f7">